src/server.rs

```diff
@@ -478,7 +491,16 @@ async fn index(
 
     debug!("OG Response : {:?}", response);
 
-    response.headers.extend(&global_response_headers);
+    response.headers.extend(&global_request_response_headers.1);
+
+    match &response_header_exclude_paths.get_ref() {
+        None => {}
+        Some(exclude_paths) => {
```

Comment:
```suggestion
        Some(exclude_paths) => {
```
something else?

Reply:
what?

Reply:
You should name `exclude_paths` differently @VishnuSanal